src/vscode-bicep/src/commands/deploy.ts

```diff
@@ -74,7 +72,9 @@ export class DeployCommand implements Command {
     }
 
     const documentPath = documentUri.fsPath;
-    const textDocument = TextDocumentIdentifier.create(documentUri.fsPath);
+    const textDocument = TextDocumentIdentifier.create(
+      encodeURIComponent(documentUri.path)
```

Comment:
This handles scenario when there are spaces in folder name.

Reply:
Good idea to add comment (spaces or other special characters)